### PR TITLE
Refactor: Related Articles - Split-up snapshots

### PR DESCRIPTION
**Pull request by aronshamash**
- Snapshot file sizes were close to `jest-lint` size limit so split by number of articles.

### CODE REVIEW

File: packages/related-articles/__tests__/shared-util.js [hunk 1]
```diff
@@ -59,6 +51,13 @@ export default ({
     mockDate.reset();
     global.Intl = realIntl;
   });
+}
+
+export const noArticlesTests = ({
```
Comment (aronshamash) [inline packages/related-articles/__tests__/shared-util.js]: Suggestions on how to split this or rename this needed.

File: packages/related-articles/__tests__/android/__snapshots__/la2-1article.android.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                <Text>
+                  <Text>
+                    
+                    Summary 125
+                    ...
+                  </Text>
+                </Text>
+              </View>
+              <Text>
+                March 13 2015, 06:54pm
```
Comment (colincclark) [inline packages/related-articles/__tests__/android/__snapshots__/la2-1article.android.test.js.snap]: is this how we do dates? looks weird

File: packages/related-articles/__tests__/shared-util.js [hunk 3]
```diff
@@ -34,15 +34,7 @@ export const createRelatedArticlesProps = (
...
-export default ({
-  fixture1,
-  fixture2,
-  fixture3,
-  one,
-  template,
-  three,
-  two
-}) => renderComponent => {
+const beforeAndAfterEach = () => {
```
Comment (colincclark) [inline packages/related-articles/__tests__/shared-util.js]: my only comment would be we should move this stuff to the test-utils iterator function. But as discussed, on another PR _a la_ https://github.com/newsuk/times-components/issues/1186

File: packages/related-articles/__tests__/android/__snapshots__/la2-1article.android.test.js.snap (hunk 2 above)
Comment (craigbilner) [reply packages/related-articles/__tests__/android/__snapshots__/la2-1article.android.test.js.snap]: now we've bumped Jest I was going to do a tech debt task to use the new date matchers so we don't have to do the funny mocking stuff

**Pull request merged**